po/af.po

```diff
@@ -0,0 +1,37 @@
+# Afrikaans translations for eknr package.
+# Copyright (C) 2018 THE eknr'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the eknr package.
+# Automatically generated, 2018.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: eknr\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-04-03 07:28+0800\n"
+"PO-Revision-Date: 2018-04-03 07:28+0800\n"
+"Last-Translator: Automatically generated\n"
+"Language-Team: none\n"
+"Language: af\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=ASCII\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: eknrenderer/eknr-renderer.c:483
+#, c-format
+msgid "This page contains content from %s, available under a %s license."
+msgstr ""
+
+#: eknrenderer/eknr-renderer.c:493
+msgid "http://wikihow.com"
```

Comment:
Do we really want this to be translated?

Reply:
If it was translated before, it should still be translated now. I think we were using this to insert things like `es.wikihow.com` in the translated messages.